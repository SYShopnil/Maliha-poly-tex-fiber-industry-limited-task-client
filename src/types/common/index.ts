export interface ICommonReturnData {
  message: string;
  status: number;
}

export enum EAuth {
  AuthTokenCookieName = "auth",
}

export enum EDataTestId {
  cLoginFormWithSubmit = "cLoginFormWithSubmit",
  cLogoutContainer = "cLogoutContainer",
  SHeaderMain = "SHeaderMain",
  SProfile = "SProfile",
}
