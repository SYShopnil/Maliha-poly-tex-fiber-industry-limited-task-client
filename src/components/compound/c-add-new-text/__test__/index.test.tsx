import { render, screen } from "@testing-library/react";
import { EDataTestId } from "@src/types/common";
import { CAddNewText } from "..";

describe("When Add New Text Component Rendered", () => {
  it("Expect render successfully", async () => {
    render(<CAddNewText />);
    const myElement = screen.getByRole(EDataTestId.CAddNewText);
    expect(myElement).toMatchSnapshot();
  });
});
