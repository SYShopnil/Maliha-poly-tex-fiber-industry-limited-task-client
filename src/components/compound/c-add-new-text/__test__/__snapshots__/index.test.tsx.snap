// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When Add New Text Component Rendered Expect render successfully 1`] = `
<div
  role="CAddNewText"
>
  <div>
    <span
      class="cursor-pointer"
      role="CButton"
    >
      <span
        class="flex justify-center items-center  "
      >
        <span
          class="text-base md:text-base_2  flex justify-center items-center px-[1rem] py-[0.5rem] rounded-[0.563rem] w-[100%] h-[100%] bg-[#EBF5EC] hover:bg-[#7F4D4F]  duration-[0.5s] text-[#7F4D4F] hover:text-[#EBF5EC]   border-[0.063rem] hover:border-[#EBF5EC] border-[#7F4D4F]   "
        >
          <span>
            Add new Text List
          </span>
        </span>
      </span>
    </span>
  </div>
  <div />
</div>
`;
